sources/packages/api/src/route-controllers/disbursement-receipt/disbursement-receipt.system-access.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { Controller, Post } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, UserToken } from "../../auth/decorators";
+import { IUserToken } from "../../auth/userToken.interface";
+import { DisbursementReceiptProcessingService } from "../../esdc-integration/disbursement-receipt-integration/disbursement-receipt-processing.service";
+import { ClientTypeBaseRoute } from "../../types";
+import { ESDCFileResponseAPIOutDTO } from "../models/esdc.shared.dto";
+import BaseController from "../BaseController";
+
+@AllowAuthorizedParty(AuthorizedParties.formsFlowBPM)
+@Controller("disbursement-receipt")
```

Comment:
This is an eCert-related operation, do we need a new controller for this one?

Reply:
I can move this to esdc-controller.